db/data_migration/20180803160419_republish_editions_with_opendocument_attachments.rb

```diff
@@ -0,0 +1,5 @@
+document_ids = Document.joins(:editions).joins("INNER JOIN attachments ON editions.id = attachments.attachable_id AND attachments.attachable_type = 'Edition'").joins("INNER JOIN attachment_data ON attachments.attachment_data_id = attachment_data.id").where("editions.state = 'published'").where("attachment_data.carrierwave_file like '%.odt' OR attachment_data.carrierwave_file like '%.odp' OR attachment_data.carrierwave_file like '%.ods'").pluck(:id).uniq
```

Comment:
Not a blocker - It's possible to make this more readable by reformatting like so:
```ruby
document_ids = Document.joins(:editions)
                       .joins("INNER JOIN attachments ON editions.id = attachments.attachable_id AND attachments.attachable_type = 'Edition'")
                       .joins("INNER JOIN attachment_data ON attachments.attachment_data_id = attachment_data.id")
                       .where("editions.state = 'published'")
                       .where("attachment_data.carrierwave_file like '%.odt' OR attachment_data.carrierwave_file like '%.odp' OR attachment_data.carrierwave_file like '%.ods'")
                       .pluck(:id).uniq

Reply:
Thanks @steventux I forgot to do this!